src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -68,7 +70,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     System.out.println("Saving about me for " + userEmail);
 
-    User user = new User(userEmail, aboutMe);
+    User userData = datastore.getUser(userEmail);
+    String profilePicUrl;
```

Comment:
Not sure if its better, but you could declare the empty string and set it in the `if` loop instead, then you wouldn't need the `else`.

Reply:
Ahh true too